packages/graphback-core/src/crud/mappingHelpers.ts

```diff
@@ -157,10 +157,17 @@ export function getInputFieldName(field: GraphQLField<any, any>): string {
   return relationshipAnnotation.key || transformForeignKeyName(field.name);
 }
 
-export function getInputFieldNamedType(schemaComposer: SchemaComposer<any>, field: GraphQLField<any, any>, operation: GraphbackOperationType): GraphQLInputType {
+export function getInputFieldNamedType(schemaComposer: SchemaComposer<any>, modelName: string, field: GraphQLField<any, any>, operation: GraphbackOperationType): GraphQLInputType {
```

Comment:
The number of parameters for this function is 4, which is a bit more than ideal. It is unavoidable to not add `modelName` here.

`schemaComposer` is passed so that this can return the input type. Another option is to return the type name from `getInputFieldNamedType` instead and let the method which called it (which would have `schemaComposer` get the input type from the name.

What you think?

Reply:
Good idea. I like it!